### PR TITLE
Add temp logging to debug SSL error.

### PR DESCRIPTION
**Pull request by millenniumiguana**
https://sentry.io/organizations/thanx/issues/2268183864/events/?project=38642&statsPeriod=14d

### CODE REVIEW

File: lib/rpush/daemon/apns2/delivery.rb [hunk 1]
```diff
@@ -17,7 +17,11 @@ def initialize(app, http2_client, batch)
 
         def perform
           @batch.each_notification do |notification|
-            prepare_async_post(notification)
+            begin
+              prepare_async_post(notification)
+            rescue OpenSSL::SSL::SSLError => error
+              log_error("Notification #{notification.id} failed with SSL error")
```
Comment (kirbycool) [inline lib/rpush/daemon/apns2/delivery.rb:23]: we still want to crash, right?

Comment (millenniumiguana) [reply lib/rpush/daemon/apns2/delivery.rb:23]: I don't think that we do, actually. As I understand it, the notifications are all mixed together from different apps. My theory for the reason we're seeing the SSL error is that we are attempting to send push notifications for an inactive or disabled app. This old app doesn't have an up to date certificate. I'd like to figure out which notifications are causing this issue but I think the rest should be allowed to go through.

Comment (kirbycool) [reply lib/rpush/daemon/apns2/delivery.rb:23]: Hmmm, that seems somewhat risky. It's ok with me if it's temporary but we should be ready to reenable cert verification asap.

Comment (millenniumiguana) [reply lib/rpush/daemon/apns2/delivery.rb:23]: Risky because we'll see this error crop up more frequently? Or something else?

Comment (millenniumiguana) [reply lib/rpush/daemon/apns2/delivery.rb:23]: Oh - this doesn't disable cert verification - apple prevents this push notification from going out anyway

Comment (kirbycool) [reply lib/rpush/daemon/apns2/delivery.rb:23]: Oh ok. Yeah, I was worried about ignoring cert verification. There's still a potential attack vector here but it's probably not a big deal in practice.

**Review by kirbycool — APPROVED** [review]

**Pull request merged**